runtime/altair/src/lib.rs

```diff
@@ -1497,11 +1497,8 @@ pub type Executive = frame_executive::Executive<
 	Runtime,
 	AllPalletsWithSystem,
 	(
-		pallet_multisig::migrations::v1::MigrateToV1<Runtime>,
-		pallet_preimage::migration::v1::Migration<Runtime>,
-		pallet_democracy::migrations::v1::Migration<Runtime>,
-		pallet_scheduler::migration::v3::MigrateToV4<Runtime>,
 		pallet_interest_accrual::migrations::v2::Migration<Runtime>,
```

Comment:
I would also remove this one. cc @branan

Reply:
Seems not to be migrated yet, or at least it migrates succesfully.

Reply:
mmh. Branan can clarify that I guess

Reply:
It will always migrate successfully, I wrote it to not fail if it was left in. It should log a warning, though.

It can safely be removed.

Reply:
Apologies, I posted wrong info.

We've definitely shipped since we landed this code, but it must have been for a security issue and thus been a minimal release instead of from `main`. Altair is still on V1 not V2. We still need this migration.

Reply:
@branan looking at the [chain state](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Faltair.api.onfinality.io%2Fpublic-ws#/chainstate) of `InterestAccrual` I see that the storage is empty. Maybe we do not need the migration